fops/background/transport/aws_lambda.py

```diff
@@ -0,0 +1,38 @@
+import json
+import logging
+import os
+from typing import Any
+
+import boto3
+from botocore.exceptions import ClientError
+
+from ..interface import ITransport, TaskEvent
+
+LOG = logging.getLogger(__name__)
+
+
+class LambdaTransport(ITransport):
+    def __init__(self, *, function_name: str = None, client: Any = None):
+        self.function_name = function_name or os.getenv("BACKGROUND_TASKS_LAMBDA_NAME")
```

Comment:
prefixed, aligned with namespace

```suggestion
        self.function_name = function_name or os.getenv("FOPS_BACKGROUND_TASKS_LAMBDA_NAME")
```